netbox/overlays/ocp-prod/deployments/netbox_config_patch.yaml

```diff
@@ -36,13 +36,63 @@ spec:
               mountPath: /etc/netbox/config/extra.py
               subPath: extra.py
         - name: netbox-worker
-          env: *env
-          envFrom: *envFrom
-          volumeMounts: *volumeMounts
+          env:
+            - name: DB_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: netbox-pguser-netbox
+                  key: host
+            - name: DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: netbox-pguser-netbox
+                  key: user
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: netbox-pguser-netbox
+                  key: password
+            - name: DB_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: netbox-pguser-netbox
+                  key: dbname
+          envFrom: &envFrom
```

Comment:
is there a reason why you kept this anchor here? `envFrom` block is created later again in this file.

Reply:
That was just a copy/paste error. Good catch, thanks!